.github/workflows/continuous-integration-and-delivery.yml

```diff
@@ -14,13 +14,21 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2.3.4
+        with:
```

Comment:
@carryall This is related to  this [comment](https://github.com/junan/nimble-survey/pull/21#discussion_r526003428). If we are ok, how about to remove it as it is not required for this `test` job?